src/main/java/cms/gov/madie/measure/services/TestCaseService.java

```diff
@@ -152,7 +152,7 @@ public TestCase updateTestCase(
     if (measure.getTestCases() == null) {
       measure.setTestCases(new ArrayList<>());
     }
-
+    measureService.verifyAuthorization(username, measureService.findMeasureById(measureId));
```

Comment:
not sure if something was missed in commit or maybe it's just not showing properly on github, but it looks like there is still an extra call to `measureService.findMeasureById(measureId)` being made within the call to `measureService.verifyAuthorization()` instead of using the `measure` fetched above